Update mosaics_cli.py

vnewey opened the pull request:
Add resampling_method as a parameter, to enable the choice of overview resampling method. NEAREST​/​AVERAGE​/​BILINEAR​/​CUBIC​/​CUBICSPLINE​/​LANCZOS​/​MODE​/​RMS

This would allow other DEA products to use it.

vnewey committed:
Update mosaics_cli.py

Add resampling_method as a parameter, to enable the choice of overview resampling method.
NEAREST​/​AVERAGE​/​BILINEAR​/​CUBIC​/​CUBICSPLINE​/​LANCZOS​/​MODE​/​RMS

vnewey committed:
Automatically update integration test validation results

robbibt commented on intertidal/mosaics.py:
```diff
@@ -87,6 +87,14 @@ def _is_s3(path):
     "Supports DEFLATE/ZSTD/LERC_DEFLATE/LERC_ZSTD/LZMA.",
 )
+@click.option(
+    "--resampling_method",
```
Can we call this "overview_resampling" to match the GDAL param? Just because GDAL also has an "r"/"resampling" param too

vnewey replied:
done. :)

vnewey committed:
Updated mosaics.py

renamed the parameter from resampling_method to overview_resampling to be consistent with the GAL parameter.
Output mosaics now have the word mosaic in their name. format {product}_mosaic_{year}_{band}.tif

vnewey committed:
pull merge

vnewey committed:
Automatically update integration test validation results

robbibt reviewed (approved):
Yay!
